pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -39,15 +39,18 @@ type Manager interface {
 
 //HintProvider interface is to be implemented by Hint Providers
 type HintProvider interface {
-	GetTopologyHints(pod v1.Pod, container v1.Container) ([]TopologyHint, bool)
+	GetTopologyHints(pod v1.Pod, container v1.Container) []TopologyHint
 }
 
 //Store interface is to allow Hint Providers to retrieve pod affinity
 type Store interface {
 	GetAffinity(podUID string, containerName string) TopologyHint
 }
 
-// TopologyHint is a struct containing Socket Mask for a Pod
+// TopologyHint is a struct containing a SocketMask for resources allocated to
+// a container and a bool indicating whether this SocketMask encodes a
```

Comment:
Could we move detailed comments on field semantics to directly precede the field declaration?

e.g.

```go
// Preferred is set when...
Preferred bool
```